jestconfig.json

```diff
@@ -19,7 +19,7 @@
     "src/index.jsx"
   ],
   "moduleNameMapper": {
-    "\\.(svg)$": "<rootDir>/__mocks__/fileMock.js",
+    "\\.(svg|gif)$": "<rootDir>/__mocks__/fileMock.js",
```

Comment:
Out of curiosity, how much effort would it take to animate an svg instead of importing a gif in the same place?

Reply:
Not exactly sure what you're getting at. This specific config is just used so that Jest, the testing framework, knows how to handle these kinds of imports. In this specific case I'm just mocking imports of svg or gif files to return an empty object.